code/tests/functional/tests/backend_api/sk_orchestrator/conftest.py

```diff
@@ -29,6 +29,9 @@ def app_config(make_httpserver, ca):
                 "AZURE_SEARCH_SERVICE": f"https://localhost:{make_httpserver.port}/",
                 "AZURE_CONTENT_SAFETY_ENDPOINT": f"https://localhost:{make_httpserver.port}/",
                 "AZURE_SPEECH_REGION_ENDPOINT": f"https://localhost:{make_httpserver.port}/",
+                "AZURE_STORAGE_ACCOUNT_ENDPOINT": f"https://localhost:{make_httpserver.port}/",
+                # TODO: Retrieve config from mocked storage with `orchestrator` set to `semantic_kernel`, instead of creating new server
```

Comment:
Nit: I try not to leave TODO's in code, they tend to get forgotten about. Can this be a task on our story instead?

Reply:
Removed, and raised #878